### PR TITLE
code for relation insersion in response

### PR DESCRIPTION
**Pull request by balfoldi**

### CODE REVIEW

**Comment by balfoldi** [comment]
Creation of helpers aiming to discharge controllers.

**Comment by balfoldi** [comment]
Tested with insomnia

File: app/models/bike.rb [hunk 1]
```diff
@@ -4,4 +4,8 @@ class Bike < ApplicationRecord
   has_one :offer
   has_many :join_table_bikes_tags
   has_many :tags, through: :join_table_bikes_tags
+
+  def api
+    self.build("owner", "spec", "tags")
```
Comment (RigateStudio) [inline app/models/bike.rb:9]: spec avec un s = specs

Comment (balfoldi) [reply app/models/bike.rb:9]: ok, merging ...

**Pull request merged**